test/controllers/patient_controller_test.rb

```diff
@@ -8,14 +8,25 @@ class PatientControllerTest < ActionDispatch::IntegrationTest
     assert_response :not_found
   end
 
-  test 'search for patients with multipe matches' do
+  test 'search for patients with multiple matches' do
     params = { given_name: 'Latrice', family_name: 'MorrisAIRA', patient_dob: '20170817' }
     VCR.use_cassette('patient_search_multi') do
       get search_path, params: params
       assert_response :unprocessable_entity
     end
   end
 
+  test 'list required and optional fields' do
+    get patient_form_path
+    assert_response :success
+  end
+
+  test 'search without required parameter' do
+    params = { given_name: 'VallisAIRA', family_name: 'RogersAIRA'}
+    get search_path, params: params
+    assert_response :unprocessable_entity
+  end
+
   test 'search for patient' do
```

Comment:
You could add testing for a patient with all info filled out, where you check to see if the params match what is returned by the patient search, similarly to the assertions on lines 46-48. This would be able to catch things like the phone being formatted incorrectly, etc.